ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/RequestResponse.kt

```diff
@@ -42,6 +42,7 @@ class Request internal constructor(
     val method: HttpMethod,
     val uri: CharSequence,
     val version: CharSequence,
+    val remoteHost: CharSequence,
```

Comment:
Not sure about it: can't we pass it somehow else? Perhaps we can pass some "client info" in `HttpRequestHandler` instead. I would prefer to group input, output and connection info in a "client info"